includes/class-mention.php

```diff
@@ -146,23 +145,4 @@ public static function get_inbox_by_mentioned_actor( $actor ) {
 
 		return new WP_Error( 'activitypub_no_inbox', \__( 'No "Inbox" found', 'activitypub' ), $metadata );
 	}
-
-	/**
-	 * Extract the mentions from the post_content.
-	 *
-	 * @param array  $mentions The already found mentions.
-	 * @param string $post_content The post content.
-	 *
-	 * @return mixed The discovered mentions.
-	 */
-	public static function extract_mentions( $mentions, $post_content ) {
-		\preg_match_all( '/@' . ACTIVITYPUB_USERNAME_REGEXP . '/i', $post_content, $matches );
-		foreach ( $matches[0] as $match ) {
-			$link = Webfinger::resolve( $match );
-			if ( ! is_wp_error( $link ) ) {
-				$mentions[ $match ] = $link;
-			}
-		}
-		return $mentions;
-	}
```

Comment:
@pfefferle Was deleting this an error?

Reply:
No, it was by intention, but not finished yet ;)

The removed version implements the same functionality as the Mention class already solves, so I planned to rewrite the Mention class a bit, to also support comments or finding mentions in texts in general.

Reply:
@pfefferle ok, probably an error then because this ^ code IS the from the Mentions class ;-)